elastic/kibana
# [Fleet] fix for upgrade package with tsds removed

## Summary

Fixes https://github.com/elastic/kibana/issues/157345

To test:

Install `nginx-1.12.0-beta` which has `index.mode:time_series`.

```
POST http://elastic:changeme@localhost:5601/api/fleet/epm/packages/nginx-1.12.0-beta
kbn-xsrf: kibana

{
   "force": true
 }
```

Upgrade to `nginx-1.12.1-beta` which has `index.mode:time_series` removed.

Upload this package built manually:
[nginx-1.12.1-beta.zip](https://github.com/elastic/kibana/files/11452945/nginx-1.12.1-beta.zip)

```
curl -XPOST -H 'content-type: application/zip' -H 'kbn-xsrf: true' http://localhost:5601/api/fleet/epm/packages -u elastic:changeme --data-binary @nginx-1.12.1-beta.zip
```

The package should install successfully and time_series should be removed from the index template `metrics-nginx.stubstatus`

WIP: update tests


### Checklist
